settings/common.py

```diff
@@ -45,6 +45,7 @@
 ]
 
 OUR_APPS = [
+    'evalai',
```

Comment:
I will suggest that this management command be kept in `base` app. Because we are actually creating confusing that `evalai` is an app. @deshraj: your input here please.